addons/pulsar/scripts/init-broker.sh

```diff
@@ -77,11 +77,11 @@ init_broker() {
   local idx=${POD_NAME##*-}
   if [ $idx -ne 0 ]; then
     wait_for_cluster_metadata "$zookeeperServers" "$clusterName"
-    echo "Cluster already initialized" && quit_script
```

Comment:
Change to `&& exit 0` to keep the semantic.

Reply:
> Change to `&& exit 0` to keep the semantic.

fixed done